xchange-core/src/main/java/org/knowm/xchange/dto/Order.java

```diff
@@ -431,6 +431,10 @@ public OrderType getOpposite() {
     REJECTED,
     /** Order has expired it's time to live or trading session and been removed from order book */
     EXPIRED,
+    /** Order is open and waiting to be filled */
+    OPEN,
+    /** Order has been either filled or cancelled */
+    CLOSED,
```

Comment:
Some of these states overlap each other, a partially filled order is technically opened, same a filled order can be closed, I am not sure if these states have not been added here before for a reason?

@timmolter

Reply:
Hello, these are the exact words that FTX sends with their response. They have only these 3